infrastructure/crowdin/setup-instance.sh

```diff
@@ -8,11 +8,14 @@ PYTHON_PACKAGE_DIR="crowdin_bot_python_package"
 
 # Install packages
 sudo apt-get install git --yes
+sudo apt-get install gettext --yes
 sudo apt-get install default-jre --yes  # Java, for crowdin cli
 sudo apt-get install python3-pip --yes
 sudo apt-get install python3-lxml --yes  # Install here instead of pip3 because compilation uses too much RAM
```

Comment:
Should these be version locked? Or are we fine as is, especially as this is not critical infrastructure.

Reply:
I've added a requirements file for the python dependencies. I assume those are the ones you meant, not the apt-get packages :)